.taskcluster.yml

```diff
@@ -181,13 +181,13 @@ tasks:
                         git fetch ${repository} refs/tags/${head_rev}
                         && git config advice.detachedHead false
                         && git checkout FETCH_HEAD
-                        && python automation/taskcluster/decision_task.py beta ${event.release.tag_name}
```

Comment:
We don't want to do betas anymore. Do I get it right?

Reply:
We still do betas, but we now know whether we're doing a beta or a release by the tag name

Reply:
Okay, let's parametrize the treeherder job symbol and the task name/desc to tell if it's a beta or an actual release.

Reply:
I just got an idea: `D-github-release`

Reply:
Discussed offline: the only way to know if a github-release is a beta or a release is to parse the tag, which can't easily be done in json-e. Mitch and I agreed to rename the symbol/job name into respectively `D-github-release` and `Fenix Github-Release Decision Task`